packages/features/tasker/tasks/translateEventTypeDescription.ts

```diff
@@ -0,0 +1,45 @@
+import { z } from "zod";
+
+import { locales as i18nLocales } from "@calcom/lib/i18n";
+import logger from "@calcom/lib/logger";
+import { EventTypeTranslationRepository } from "@calcom/lib/server/repository/eventTypeTranslation";
+
+export const ZTranslateEventTypeDescriptionPayloadSchema = z.object({
+  eventTypeId: z.number(),
+  userId: z.number(),
+  description: z.string(),
+  userLocale: z.string(),
+});
+
+export type TranslateEventTypeDescriptionPayload = z.infer<
+  typeof ZTranslateEventTypeDescriptionPayloadSchema
+>;
+
+export async function translateEventTypeDescription(payload: string): Promise<void> {
+  try {
+    const { eventTypeId, description, userLocale, userId } =
+      ZTranslateEventTypeDescriptionPayloadSchema.parse(JSON.parse(payload));
+
+    const targetLocales = (["en", "es", "de", "pt", "fr", "it", "ar", "ru", "zh-CN"] as const).filter(
+      (locale) => locale !== userLocale && i18nLocales.includes(locale)
+    );
+    const { ReplexicaService } = await import("@calcom/lib/server/service/replexica");
```

Comment:
dynamic importing is needed, otherwise we get an error complaining about packages replexica sdk use as dependencies